pkgs/development/python-modules/groq/default.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  hatch-fancy-pypi-readme,
+  hatchling,
+  anyio,
+  cached-property,
+  distro,
+  httpx,
+  pydantic,
+  sniffio,
+  typing-extensions,
+}:
+
+buildPythonPackage rec {
+  pname = "groq";
+  version = "0.11.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-27mu/t84jd1IAex78+un9e22eUj+wM0oKdlyRAWfQqc=";
+  };
+
+  build-system = [
+    hatch-fancy-pypi-readme
+    hatchling
+  ];
+
+  dependencies = [
+    anyio
+    cached-property
+    distro
+    httpx
+    pydantic
+    sniffio
+    typing-extensions
+  ];
+
+  pythonImportsCheck = [
+    "groq"
+  ];
+
```

Comment:
Could you enable tests with `pytestCheckHook`?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pytestcheckhook-using-pytestcheckhook

Reply:
All the tests are heavily network-dependent, so I don't think they'd be usefull

Reply:
Well, almost all tests..I'll see what I can do